konishi_utils.py

```diff
@@ -0,0 +1,116 @@
+import time
+import hashlib
+import base64
+import models
+import json
+from inspect import isclass
+from app import *
+hash_length = 32
+
+#Self explanatory
+def get_time():
+    return int(time.time())
+
+#Concatenates all the parameters except length into a string and hashes it with sha256
+def make_hash(*args,length=hash_length):
+    tobehashed = ''
+    for arg in args:
+        tobehashed = tobehashed+str(arg)
+    tobehashed = tobehashed.encode("utf-8")
+    hash_object = hashlib.sha256(tobehashed)
+    return hash_object.hexdigest()[:length]
+
+#Checks if email address is valid. Checking is provided by hunter.io. Needs an API key to work.
+def valid_email(email):
+    res = hunter.email_verifier(email)
+    if res['gibberish']==False and res['disposable']==False and res['regexp']==True and res['score']>=50:
```

Comment:
I don't think there's much value in using an external API just to verify whether an email is legit. I don't think we should prevent people from using disposable emails if that's what they want to do. Protection against spambots can be done with captchas instead.

Reply:
Hmm understood